src/palace.pyx

```diff
@@ -143,22 +143,24 @@ def current_context(thread: bool = False) -> Optional[Context]:
 def use_context(context: Optional[Context], thread: bool = False) -> None:
     """Make the specified context current for OpenAL operations.
 
-    If `thread` is set to `True`,  it will use a context thread.
+    Default value for `thread` is `False`. If `thread` is set to `True`,
+    make the specified context current for OpenAL operations on the calling
+    thread only. Requires the `ALC_EXT_thread_local_context` extension on
+    both the context's device and the DeviceManager.
 
     See Also
     --------
     Context : Audio environment container
     """
+    if context is None:
+        al_context = <alure.Context> nullptr
+    else:
+        al_context = (<Context> context).impl
```

Comment:
```suggestion
    cdef alure.Context alure_context = nullptr
    if context is not None:
        alure_context = (<Context> context).impl
```

I'd feel much safer with C++ objects being typed.  Please don't apply the patch using GitHub though since It might not work.